Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -21,8 +22,13 @@
 #include <QVBoxLayout>
 #include <WhiteBox/EditorWhiteBoxComponentBus.h>
 
+#include <AzCore/Settings/SettingsRegistry.h>
 namespace WhiteBox
 {
+    constexpr AZStd::string_view WhiteBoxTransformFeature =
+        "/O3DE/Preferences/WhiteBox/TransformFeature";
```

Comment:
uber nit: Can this just be on one line (line limit is 140 characters and this is only 107 when all on one line